test/scenarios/el94-src@low-latency.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export SKIP_GREENBOOT=true
+export TEST_RANDOMIZATION=none
```

Comment:
Are there dependencies in tests running order?

Reply:
Yes, tests are meant to be ran in order. I didn't wanted to jam all the stuff into a single test case, so I made them smaller.